appveyor.yml

```diff
@@ -0,0 +1,14 @@
+version: 1.0.{build}
```

Comment:
Well... what happens if we realize that we actually want to use a real version number and we've already made several builds named 1.0? Feels like `0.1.{build]` would be a better idea. Is `{build}` even a valid version number?